app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,264 @@
+                <% bio_class << " is-valid" %>
+              <% end %>
+            <% end %>
+
+            <%= f.label :bio %>
+
+            <%= f.text_area :bio, :class => bio_class, :placeholder => "Enter bio" %>
+
+            <% if bio_was_invalid %>
+              <% resource.errors.full_messages_for(:bio).each do |message| %>
+                <div class="invalid-feedback d-flex">
+                  <%= message %>
+                </div>
+              <% end %>
+            <% end %>
+          </div>
+          <!-- Devise Input for bio end -->
+
+          <!-- Devise Input for preferred_timezone start -->
+          <div class="form-group">
+            <% preferred_timezone_was_invalid = resource.errors.include?(:preferred_timezone) %>
+
+            <% preferred_timezone_class = "form-select" %>
+
+            <% if was_validated %>
+              <% if preferred_timezone_was_invalid %>
+                <% preferred_timezone_class << " is-invalid" %>
+              <% else %>
+                <% preferred_timezone_class << " is-valid" %>
+              <% end %>
+            <% end %>
+
+            <%= f.label :preferred_timezone %>
+
+            <%= f.select :preferred_timezone, User.preferred_timezones.keys.map{ |key| [key.upcase, key] }, { selected: @user.preferred_timezone|| :draft }, { :class => preferred_timezone_class, :placeholder => "Enter your preferred timezone" } %>
+
+            <% if preferred_timezone_was_invalid %>
+              <% resource.errors.full_messages_for(:preferred_timezone).each do |message| %>
+                <div class="invalid-feedback d-flex">
+                  <%= message %>
+                </div>
+              <% end %>
+            <% end %>
+          </div>
+          <!-- Devise Input for preferred_timezone end -->
+
+          <div class="form-group">
+            <% current_password_was_invalid = resource.errors.include?(:current_password) %>
+
+            <% current_password_class = "form-control" %>
+
+            <% if was_validated %>
+              <% if current_password_was_invalid %>
+                <% current_password_class << " is-invalid" %>
+              <% else %>
+                <% current_password_class << " is-valid" %>
+              <% end %>
+            <% end %>
+
+            <%= f.label :current_password %>
+
+            <%= f.password_field :current_password, class: current_password_class, autocomplete: "off" %>
+
+            <% if current_password_was_invalid %>
+              <% resource.errors.full_messages_for(:current_password).each do |message| %>
+                <div class="invalid-feedback d-flex">
+                  <%= message %>
+                </div>
+              <% end %>
+            <% end %>
+
+            <small class="form-text text-muted">
+              Current password is required to confirm any changes.
+            </small>
+          </div>
+
+
+          <%= f.button class: "btn btn-outline-primary btn-block" %>
```

Comment:
Same comment here about button name. Currently is 'Update User' maybe should just say 'Update'

Reply:
See commits: 
c866a63- UI changes based off Amanda's feedback
c2bdaf9- Layout change for create pair button